src/go/k8s/config/rbac/role.yaml

```diff
@@ -95,6 +95,7 @@ rules:
   - delete
   - get
   - list
+  - update
```

Comment:
NIT: This change should go in previous commit https://github.com/redpanda-data/redpanda/pull/6942/commits/827d7ea0dde0b16d740be329fba24c0c35b3eacc as it was added there.

Reply:
moved